hw3/src/Styles.css

```diff
@@ -0,0 +1,106 @@
+.App {
+  font-family: sans-serif;
+  text-align: center;
+}
+
+.container {
+  text-align: center;
+}
+
+.home {
+  text-align: center;
+  font-family: sans-serif;
+  margin-top: 3em;
+}
+
+.navbar {
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  font-family: sans-serif;
+  background-color: lightblue;
+}
+
+.navbar ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.navbar ul li {
+  display: inline-block;
+}
+
+.navbar ul li a {
+  display: block;
+  width: 100%;
+  padding: 15px 1em;
+  text-decoration: none;
+  font-weight: lighter;
+  color: #333;
+}
+
+.navbar ul li a:hover {
```

Comment:
When hovering, the background color highlights over the other navigation tabs. It overlaps